dotnet/src/Connectors/Connectors.AI.OpenAI/ChatCompletionWithData/AzureOpenAIChatCompletionWithDataService.cs

```diff
@@ -79,7 +79,7 @@ public async Task<IReadOnlyList<TextContent>> GetTextContentsAsync(string prompt
     {
         await foreach (var streamingChatContent in this.InternalGetChatStreamingContentsAsync(new ChatHistory(prompt), executionSettings, kernel, cancellationToken).ConfigureAwait(false))
         {
-            yield return new StreamingTextContent(streamingChatContent.Content, streamingChatContent.ChoiceIndex, streamingChatContent, streamingChatContent.ModelId, Encoding.UTF8, streamingChatContent.Metadata);
+            yield return new StreamingTextContent(streamingChatContent.Content, streamingChatContent.ChoiceIndex, streamingChatContent.Content, streamingChatContent.ModelId, Encoding.UTF8, streamingChatContent.Metadata);
```

Comment:
Spotted a bug in here (modelId and innerobject arguments are inverted), I will create UT to ensure we are using the correct arguments for all contents.

For same reasons above, we need to pass the streamingChat as the innerobject.
```suggestion
            yield return new StreamingTextContent(streamingChatContent.Content, streamingChatContent.ChoiceIndex, streamingChatContent.ModelId, streamingChatContent, Encoding.UTF8, streamingChatContent.Metadata);
```